templates/admin/console.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+
+{% block title %}Admin Console{% endblock %}
+
+{% block main %}
+<h1>Admin Console</h1>
+<a class="btn btn-primary" href="/admin/submissions" role="button">Submissions</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/users" role="button">Users</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createannouncement" role="button">Create Announcement</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createcontest" role="button">Create Contest</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createproblem" role="button">Create Problem</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/problems/draft" role="button">Draft Problems</a>
+<hr>
```

Comment:
I think you should add an indicator as to whether maintenance mode is on or off. Something like
`<p>Maintenance mode is currently {{ 'enabled' if maintenance else 'disabled' }}.</p>`
and then when rendering the template, change the code to like
```py
global maintenance_mode
return render_template('admin/console.html', maintenance=maintenance_mode)
```